_candidates/dan-kalb.md

```diff
@@ -92,9 +93,7 @@ expenditures_by_type:
   'Information Technology Costs (Internet, E-mail)': 4226.01
   Independent Expenditure Supporting/Opposing Others: 400
   Transfer Between Committees of the Same Candidate/sponsor: 13500
-ballots:
-- _ballots/oakland/2016-11-08.md
```

Comment:
Should this have been removed? This seems wrong. I only ran `gulp pull`.

Reply:
Right, you don't want to remove this. The gulp commands are dumb, they don't merge with the existing files, so you have to manually check for these and exclude them from the diff. I use `git add -p` for this.